src/NuGet.Core/NuGet.Packaging/ContentModel/ContentItem.cs

```diff
@@ -7,12 +7,25 @@ namespace NuGet.ContentModel
 {
     public class ContentItem
     {
+        private Dictionary<string, object> _properties;
         public string Path { get; set; }
-        public IDictionary<string, object> Properties { get; } = new Dictionary<string, object>();
+
+        public Dictionary<string, object> Properties
+        {
+            get => _properties ?? CreateDictionary();
```

Comment:
I guess every repo/team/project has different guidelines :) I was just curious if there is a reason not to use the pattern I personally find quite nice.